plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -170,6 +171,13 @@ public static function get_plugin( $slug ) {
 				'manage_url'     => 'admin.php?page=mailchimp-woocommerce',
 				'is_built_by_wc' => false,
 			),
+			'klaviyo'         => array(
+				'name'           => __( 'Klaviyo', 'woocommerce' ),
+				'description'    => __('Grow and retain customers with intelligent, impactful email and SMS marketing automation and a consolidated view of customer interactions.', 'woocommerce' ),
```

Comment:
```suggestion
				'description'    => __( 'Grow and retain customers with intelligent, impactful email and SMS marketing automation and a consolidated view of customer interactions.', 'woocommerce' ),
```

Reply:
Thanks @JPry ! I have fixed the PHP linting error (line 174 and line 176) in [`e004f90` (#39087)](https://github.com/woocommerce/woocommerce/pull/39087/commits/e004f90c3f84106b025530df96728ae8e8a6495e).